src/whylogs/app/dataset.py

```diff
@@ -0,0 +1,81 @@
+import os
+from typing import Any, Callable, List, Optional, Tuple
+import ABC
+
+from PIL import
+from PIL.TiffTags import TAGS
+
+EXTENSIONS = ('.csv','.xls','.jpg', '.jpeg', '.png', '.ppm', '.bmp', 
+              '.pgm', '.tif', '.tiff','.webp')
+
+
+
+class Dataset(ABC):
+"""
+"""
+    def __init__(self,
+        root_folder: str = "",
+        transforms : Optional[Callable]=None,
+        )->None:
+    def __getitem__(self, index: int) -> Any:
+        raise NotImplementedError
+    def __len__(self)-> int :
+        raise NotImplementedError
+
+    def __repr__(self)-> str:
+        head = "Dataset "+ self.__class__.__name__
+        body = ["Number of files: {}".format(self.__len__())]
+        if self.root is not None:
+            body.append("Folder location: {}".format(self.root))
+        body += self.extra_repr().splitlines()
+        if hasattr(self, "transforms") and self.transforms is not None:
+            body += [repr(self.transforms)]
+        lines = [head] + [" " * self._repr_indent + line for line in body]
+        return '\n'.join(lines)
+
+
+class WhylogsDataset(Dataset):
```

Comment:
we only use `whylogs` as lower case - it's an explicit decision :)

Reply:
This is a class name, which by pep8 standards needs to be uppercase

Reply:
will modify name